src/ApiGateways/Web.Bff.Shopping/aggregator/Services/OrderApiClient.cs

```diff
@@ -1,40 +1,31 @@
-﻿using Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Config;
-using Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Models;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Options;
-using System.Net.Http;
-using System.Threading.Tasks;
-using System.Text.Json;
+﻿namespace Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Services;
 
-namespace Microsoft.eShopOnContainers.Web.Shopping.HttpAggregator.Services
+public class OrderApiClient : IOrderApiClient
 {
-    public class OrderApiClient : IOrderApiClient
-    {
-        private readonly HttpClient _apiClient;
-        private readonly ILogger<OrderApiClient> _logger;
-        private readonly UrlsConfig _urls;
+    private readonly HttpClient _apiClient;
+    private readonly ILogger<OrderApiClient> _logger;
+    private readonly UrlsConfig _urls;
 
-        public OrderApiClient(HttpClient httpClient, ILogger<OrderApiClient> logger, IOptions<UrlsConfig> config)
-        {
-            _apiClient = httpClient;
-            _logger = logger;
-            _urls = config.Value;
-        }
+    public OrderApiClient(HttpClient httpClient, ILogger<OrderApiClient> logger, IOptions<UrlsConfig> config)
+    {
+        _apiClient = httpClient;
+        _logger = logger;
+        _urls = config.Value;
+    }
 
-        public async Task<OrderData> GetOrderDraftFromBasketAsync(BasketData basket)
-        {
-            var url = _urls.Orders + UrlsConfig.OrdersOperations.GetOrderDraft();
-            var content = new StringContent(JsonSerializer.Serialize(basket), System.Text.Encoding.UTF8, "application/json");
-            var response = await _apiClient.PostAsync(url, content);
+    public async Task<OrderData> GetOrderDraftFromBasketAsync(BasketData basket)
+    {
+        var url = _urls.Orders + UrlsConfig.OrdersOperations.GetOrderDraft();
```

Comment:
```suggestion
        var url = $"{_urls.Orders}{UrlsConfig.OrdersOperations.GetOrderDraft()}";
```

Interpolated strings over concatenation.

Reply:
Yes, makes sense.